File: PAT (Basic-Level)/1026.c
#include <stdio.h>
#include <time.h>

int main()
{
	long int C1, C2;
	scanf("%ld %ld", &C1, &C2);
	float time = (float) ((C2 - C1) / 100.0);
	int hour, min, second;
	int se_time;
	if((int)(time * 10) % 10 > 4){
		se_time = (int)time;
		se_time++;
	}else if((int)(time * 10) % 10 <= 4){
		se_time = (int)time;
	}
	hour = se_time / 3600;
	min = (se_time - 3600 * hour) / 60;
	second = se_time - 3600 * hour - min * 60;
	printf("%.2d:%.2d:%.2d", hour, min, second);
	return 0;
}

File: PAT (Basic-Level)/Python/1026.py
#!/usr/bin/python3
# -*- coding: utf-8 -*-

timeSet = list(map(int, input().split()))
time = ((timeSet[1] - timeSet[0]) / 100.0)

if int((time * 10) % 10) > 4:
    se_time = int(time)
    se_time += 1
elif int((time * 10) % 10) <= 4:
    se_time = int(time)

hour = int(se_time / 3600)
min = int((se_time - 3600 * hour) / 60)
second = int(se_time - 3600 * hour - min * 60)
print(hour)
print(min)
print(second)
print("%.2d:%.2d:%.2d" % (hour, min, second))
